pwndbg/commands/heap.py

```diff
@@ -371,3 +371,89 @@ def find_fake_fast(addr, size):
 
             if main_heap.fastbin_index(value) == fastbin:
                 malloc_chunk(start+offset-pwndbg.arch.ptrsize,fake=True)
+
+
+@pwndbg.commands.ParsedCommand
+@pwndbg.commands.OnlyWhenRunning
+@pwndbg.commands.OnlyWhenHeapIsInitialized
+def vis_heap_chunks(addr, final_count=2):
+    """
+    Visualize heap chunks at the specified address
+    """
+    addr = int(addr)
+    main_heap = pwndbg.heap.current
+    main_arena = main_heap.get_arena()
+    top_chunk = int(main_arena['top'])
+
+    fmt = {
+        'little': '<',
+        'big': '>'
+    }[pwndbg.arch.endian] + {
+        4: 'I',
+        8: 'Q'
+    }[pwndbg.arch.ptrsize]
+
+    def unpack(x): return struct.unpack(fmt, x)[0]
```

Comment:
We can probably remove the formatting code and just use `pwndbg.arch.unpack(...)`